moveit_core/trajectory_processing/include/moveit/trajectory_processing/time_optimal_trajectory_generation.h

```diff
@@ -189,6 +189,13 @@ class Trajectory
   mutable std::list<TrajectoryStep>::const_iterator cached_trajectory_segment_;
 };
 
+/**
+ * @brief Converts a `trajectory_processing::Trajectory` into a `JointTrajectory` message with a given sampling rate.
+ */
+[[nodiscard]] trajectory_msgs::msg::JointTrajectory
+createTrajectoryMessage(const std::vector<std::string>& joint_names,
+                        const trajectory_processing::Trajectory& trajectory, const int sampling_rate);
```

Comment:
Can this free function be moved to `trajectory_tool.h` file so that its easier to find and not get lost among other class declarations.